spec/factories/picture_phenotypes.rb

```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
+  factory :picture_phenotype do
+    characteristic "Eye color"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.